mantle/kola/harness.go

```diff
@@ -734,16 +783,17 @@ ExecStart=%s
 
 		AdditionalDisks: targetMeta.AdditionalDisks,
 		MinMemory:       targetMeta.MinMemory,
+		NonExclusive:    !targetMeta.Exclusive,
```

Comment:
```suggestion
		Exclusive:    targetMeta.Exclusive,
```

For some reason I would still just use `Exclusive` right here and in the `type Test struct` and then all other uses of `nonExclusive` seem fine.

Reply:
If we change the field to `Exclusive` we would have to enforce the default value to be `true`. It doesn't look like golang has constructors that we can use to enforce default values. Is there some other way this can be done?

Reply:
Apparently not, without creating our own constructor.. OK, let's leave this like it is.